src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public static class BitConverterTests
+{
+    [Fact]
+    public static void DoubleToInt64Bits()
+    {
+        Double input = 123456.3234;
```

Comment:
Rather than doing this manually, if we want to do this we should add a rule to the [codeformatter](https://github.com/dotnet/codeformatter/tree/master/src/Microsoft.DotNet.CodeFormatting/Rules) and run it on the whole repository.